System.IO.Abstractions.TestingHelpers.Tests/MockFileInfoTests.cs

```diff
@@ -536,6 +536,28 @@ public void MockFileInfo_ToString_ShouldReturnOriginalFilePath(string path)
             Assert.AreEqual(filePath, mockFileInfo.ToString());
         }
 
+
+        /// <summary>
+        /// Normalize, tested with Path.GetFullPath and new FileInfo().FullName;
+        /// </summary>
+        [WindowsOnly(WindowsSpecifics.StrictPathRules)]
+        [TestCase(@"c:\top\..\most\file", @"c:\most\file")]
```

Comment:
Is this really Windows-specific?

Reply:
Well drives are Windows only? Not sure how to setup a non-windows test (duplicate test?) and if it's really needed, those will cover the same lines